ledger-core/v2/application/functest/api_test_utils.go

```diff
@@ -28,8 +28,10 @@ const (
 	requestTimeout = 30 * time.Second
 	contentType    = "Content-Type"
 
-	defaultHost      = "127.0.0.1"
-	createWalletPath = "/wallet/create"
+	defaultHost          = "127.0.0.1"
+	walletCreatePath     = "/wallet/create"
```

Comment:
I would move "/wallet" to separate variable and reuse it. But it's not necessary.

Reply:
done